handle slides that do not have spacing information

i got this error:

```python
Traceback (most recent call last):
  File "/mnt/data08/shared/jkaczmarzyk/patch-classification-pipeline/venv/bin/wsi_run", line 33, in <module>
    sys.exit(load_entry_point('wsi-inference', 'console_scripts', 'wsi_run')())
  File "/mnt/data08/shared/jkaczmarzyk/patch-classification-pipeline/venv/lib/python3.9/site-packages/click/core.py", line 1130, in __call__
    return self.main(*args, **kwargs)
  File "/mnt/data08/shared/jkaczmarzyk/patch-classification-pipeline/venv/lib/python3.9/site-packages/click/core.py", line 1055, in main
    rv = self.invoke(ctx)
  File "/mnt/data08/shared/jkaczmarzyk/patch-classification-pipeline/venv/lib/python3.9/site-packages/click/core.py", line 1404, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/mnt/data08/shared/jkaczmarzyk/patch-classification-pipeline/venv/lib/python3.9/site-packages/click/core.py", line 760, in invoke
    return __callback(*args, **kwargs)
  File "/mnt/data08/shared/jkaczmarzyk/patch-classification-pipeline/venv/lib/python3.9/site-packages/click/decorators.py", line 26, in new_func
    return f(get_current_context(), *args, **kwargs)
  File "/mnt/data08/shared/jkaczmarzyk/patch-classification-pipeline/wsi_inference/main.py", line 212, in cli
    run_inference(
  File "/mnt/data08/shared/jkaczmarzyk/patch-classification-pipeline/wsi_inference/modellib/run_inference.py", line 240, in run_inference
    dset = WholeSlideImagePatches(
  File "/mnt/data08/shared/jkaczmarzyk/patch-classification-pipeline/wsi_inference/modellib/run_inference.py", line 120, in __init__
    assert pathlib.Path(patch_path).exists(), "patch path not found"
AssertionError: patch path not found
```

--- a/wsi_inference/modellib/run_inference.py
+++ b/wsi_inference/modellib/run_inference.py
@@ -206,7 +206,7 @@ def run_inference(
     if patch_paths_notfound:
         warnings.warn(
             "Patch extraction seems to have failed for the following slides:"
-            + " ".join(str(p) for p in patch_paths_notfound),
+            + " ".join(str(p.stem) for p in patch_paths_notfound),
             category=PatchFilesNotFoundWarning,
         )
 
@@ -235,6 +235,10 @@ def run_inference(
         if slide_csv.exists():
             print("Output CSV exists... skipping.")
             print(slide_csv)
+            continue
+
+        if not patch_path.exists():
+            print(f"Skipping because patch file not found: {patch_path}")
             continue
 
         dset = WholeSlideImagePatches(

